llvm/lib/IR/LLVMContext.cpp

```diff
@@ -31,6 +31,35 @@
 
 using namespace llvm;
 
+static constexpr StringRef knownBundleName(unsigned BundleTagID) {
+  switch (BundleTagID) {
+  case LLVMContext::OB_deopt:
+    return "deopt";
+  case LLVMContext::OB_funclet:
+    return "funclet";
+  case LLVMContext::OB_gc_transition:
+    return "gc-transition";
+  case LLVMContext::OB_cfguardtarget:
+    return "cfguardtarget";
+  case LLVMContext::OB_preallocated:
+    return "preallocated";
+  case LLVMContext::OB_gc_live:
+    return "gc-live";
+  case LLVMContext::OB_clang_arc_attachedcall:
+    return "clang.arc.attachedcall";
+  case LLVMContext::OB_ptrauth:
+    return "ptrauth";
+  case LLVMContext::OB_kcfi:
+    return "kcfi";
+  case LLVMContext::OB_convergencectrl:
+    return "convergencectrl";
+  default:
+    return "";
```

Comment:
Minor nit: can this be `llvm_unreachable`?

Reply:
Yes, as it is now. I was debating surfacing this to a public query for the name but I don't know if it would be of any use